src/bin/xmoji/unistr.h

```diff
@@ -4,7 +4,11 @@
 #include "macros.h"
 
 #include <poser/decl.h>
-#include <uchar.h>
+#include <stddef.h>
```

Comment:
I'll try to move this to a separate compatibility header that still uses `uchar.h` when available.

Reply:
Done in commit [a28946b](https://github.com/Zirias/xmoji/commit/a28946b23e013f3c596b801a5c6f1a75c2000c90), could you please double-check?

Reply:
You missed a spot :)
tools/emojigen/src/emojigen.c uses uchar.h too.

Reply:
thanks! Will fix and then add to master. I'll also pull the statvfs.